can-control.js

```diff
@@ -285,14 +284,13 @@ var Control = Construct.extend(
 
 			if(!element) {
 				dev.warn('can/control/control.js: Creating an instance of a named control without passing an element');
-
-				element = vDom().createElement('div');
+				return;
```

Comment:
I agree. I didn't like that we are just returning with a warning. I think throw would be a better developer experience.